docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3.9"
```

Comment:
This is the docker-compose version. 3.9 is not released yet. This should be left as 3.7 for now.